src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -76,57 +80,50 @@ export default class GeneBarPlot extends React.Component<IGeneBarPlotProps, {}>
     }
 
     @autobind
-    @action
-    private onPlotTypeSelect(option: any) {
-        this.plotType = option.value
-    }
+    private getTooltip(datum: any) {
+        let geneSymbol = datum.majorCategory as string;
+        //remove * from end of gene symbol if any
+        if (geneSymbol.endsWith("*")) {
```

Comment:
unfortunately IE11 doesn't support endsWith and it's not polyfilled as far as I can see.  Probably simplest just to use /*?/.test() for now

Reply:
actually, instead of testing for this you could just do a one liner.  geneSymbol = genSymbo.replace(/*$/,"")